notary/notary.spec

```diff
@@ -0,0 +1,310 @@
+Version:        0.1
+Release:        1.20160219git%{shortcommit}%{?dist}
+Summary:        A server and client for running and interacting with trusted collections
+License:        ASL 2.0
+URL:            https://%{provider_prefix}
+Source0:        https://%{provider_prefix}/archive/%{commit}/%{repo}-%{shortcommit}.tar.gz
+Source1:        notary-server.service
+Source2:        notary-signer.service
+# Edits example config files to make it easier to start with Notary, but the
+# edits also (intentionally) prevent running with unmodified configuration;
+# OTOH upstream wants the unmodified configuration to work in docker-compose
+# setups. So, not sent upstream.
+Patch0:         notary-config.patch
+
+
+# e.g. el6 has ppc64 arch without gcc-go, so EA tag is required
+ExclusiveArch:  %{?go_arches:%{go_arches}}%{!?go_arches:%{ix86} x86_64 %{arm}}
+# If go_compiler is not set to 1, there is no virtual provide. Use golang instead.
+BuildRequires:  %{?go_compiler:compiler(go-compiler)}%{!?go_compiler:golang}
+BuildRequires: libtool-ltdl-devel systemd
+Requires(post): systemd
+Requires(preun): systemd
+Requires(postun): systemd
+
+%description
+%{summary}
+
+%post
+%systemd_post notary-server.service notary-signer.service
+
+%preun
+%systemd_preun notary-server.service notary-signer.service
+
+%postun
+%systemd_postun_with_restart notary-server.service notary-signer.service
+
+%if 0%{?with_devel}
+%package devel
+Summary:       %{summary}
+BuildArch:     noarch
+
+%if 0%{?with_check} && ! 0%{?with_bundled}
+BuildRequires: golang(github.com/Sirupsen/logrus)
+BuildRequires: golang(github.com/Sirupsen/logrus/hooks/bugsnag)
+BuildRequires: golang(github.com/agl/ed25519)
+BuildRequires: golang(github.com/bugsnag/bugsnag-go)
+BuildRequires: golang(github.com/docker/distribution/context)
+BuildRequires: golang(github.com/docker/distribution/health)
+BuildRequires: golang(github.com/docker/distribution/registry/api/errcode)
+BuildRequires: golang(github.com/docker/distribution/registry/api/v2)
+BuildRequires: golang(github.com/docker/distribution/registry/auth)
+BuildRequires: golang(github.com/docker/distribution/registry/auth/silly)
+BuildRequires: golang(github.com/docker/distribution/uuid)
+BuildRequires: golang(github.com/docker/docker/pkg/term)
+BuildRequires: golang(github.com/docker/go-connections/tlsconfig)
+BuildRequires: golang(github.com/docker/go/canonical/json)
+BuildRequires: golang(github.com/dvsekhvalnov/jose2go)
+BuildRequires: golang(github.com/go-sql-driver/mysql)
+BuildRequires: golang(github.com/golang/protobuf/proto)
+BuildRequires: golang(github.com/google/gofuzz)
+BuildRequires: golang(github.com/gorilla/mux)
+BuildRequires: golang(github.com/jinzhu/gorm)
+BuildRequires: golang(github.com/mattn/go-sqlite3)
+BuildRequires: golang(github.com/miekg/pkcs11)
+BuildRequires: golang(github.com/prometheus/client_golang/prometheus)
+BuildRequires: golang(github.com/spf13/viper)
+BuildRequires: golang(github.com/stretchr/testify/assert)
```

Comment:
golang(github.com/stretchr/testify/assert) can be removed, it is required by tests only